development/markdown-index/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.19.3-alpine3.16 as builder
+
+WORKDIR /go/src/github.com/kyma-project/test-infra
+COPY . .
+
+RUN  CGO_ENABLED=0 go build -o /markdown-index -ldflags="-s -w" ./development/markdown-index
+
+FROM alpine:3.16.3
```

Comment:
Also consider adding dependabot configuration for automatic bumps

Reply:
done